docs/templates/release-notes-regular.md

```diff
@@ -0,0 +1,11 @@
+This release includes various new features, stability improvements, and security fixes. If you are on a Mac or PC you should also update Docker Desktop for security reasons.
+
+The highlights for this release are:
+  - **<!--Highlight 1-->**
+    <!-- highlight description (issue link) -->
+  - **<!--Highlight N-->**
+    <!-- highlight description (issuegit link)-->
```

Comment:
I would propose converting all the above comments to actual text, so that people can look at the rendered Markdown from the GitHub UI, without having to check the raw Markdown file.

Also, I propose having examples in this list like:

```suggestion
  - **Important accomplishment**
    We used to do [this](https://github.com/freedomofpress/dangerzone/issues/1), but now we do [that](https://github.com/freedomofpress/dangerzone/issues/2).
  - **Support for a new platform**
    We added support for a new platform ([#3](https://github.com/freedomofpress/dangerzone/issues/3))
```

This way, it's easy to recall what tense do we use, how we link to issues, etc.